run/multiple_regions/main.tf

```diff
@@ -102,7 +102,7 @@ resource "google_compute_managed_ssl_certificate" "lb_default" {
   name     = "myservice-ssl-cert"
 
   managed {
-    domains = [var.domain_name]
+    domains = ["example.com"]
```

Comment:
lint: you need to also remove the variable declaration

Reply:
Provided a code suggestion below, please check & apply if it works for you.